advanced/Scripts/webpage.sh

```diff
@@ -211,6 +211,10 @@ SetExcludeClients(){
 
 }
 
+Halt(){
+        nohup bash -c "sleep 5; halt" &> /dev/null </dev/null &
```

Comment:
Hmm... Don't think I can specify a delay in seconds to "shutdown"...  "+m" specifies it in minutes. Do we really want to add a one-minute delay? Using "+0" or "now" would be immediate, so the message is perhaps of limited value then.